python/pyspark/sql/tests/arrow/test_arrow_map.py

```diff
@@ -246,14 +246,22 @@ def setUpClass(cls):
         cls.spark.conf.set("spark.sql.execution.arrow.maxBytesPerBatch", "10")
 
 
-class MapInArrowWithOutputArrowBatchSlicingTests(MapInArrowTests):
+class MapInArrowWithOutputArrowBatchSlicingRecordsTests(MapInArrowTests):
     @classmethod
     def setUpClass(cls):
         MapInArrowTests.setUpClass()
         cls.spark.conf.set("spark.sql.execution.arrow.maxRecordsPerBatch", "10")
         cls.spark.conf.set("spark.sql.execution.arrow.maxRecordsPerOutputBatch", "3")
 
 
+class MapInArrowWithOutputArrowBatchSlicingBytesTests(MapInArrowTests):
+    @classmethod
+    def setUpClass(cls):
+        MapInArrowTests.setUpClass()
+        cls.spark.conf.set("spark.sql.execution.arrow.maxRecordsPerBatch", "10")
```

Comment:
Just a question. Do we still need this `maxRecordsPerBatch=10` for `Bytes` test?